### DISCUSSION TITLE
Bullseye ARM64 Release - 4.4.0

### DISCUSSION THREAD

**Discussion by mcguirepr89** [Motion-Project/motion#1560] [Q&A] [answered]
Hello, I am curious if there is an `arm64` release for motion 4.4.0 on Bullseye? I noticed there seems to be an `arm64` version for Buster, but wanted to see about one for Bullseye.

Thank you for all of your work on this project and thank you in advance to whoever it is that answers this question.

My very best regards,
Patrick

**Comment by tosiara** [comment, marked as answer]
Hi,
there was an issue with Bullseye's kernel and `qemu-user-static`+`docker` at the time of release which caused segfault during installation of mariadb library inside docker:
```
Processing triggers for libc-bin (2.31-11) ...
qemu: uncaught target signal 11 (Segmentation fault) - core dumped
```
Because of that arm64 package did not get built at that time.

Now it seems to be resolved and will try to update the release soon

**Reply by mcguirepr89** [reply]
Thank you kindly for the swift and informative reply! I will keep an eye out for the build and will use it when it is ready ~~

My very best,
Patrick

**Comment by tosiara** [comment]
Please check: https://github.com/Motion-Project/motion/releases/tag/release-4.4.0
Also note, that this arm64 build does not contain raspberry camera support
